src/libcore/fmt/builders.rs

```diff
@@ -190,7 +195,7 @@ impl<'a, 'b: 'a> DebugStruct<'a, 'b> {
     /// }
     ///
     /// assert_eq!(
-    ///     format!("{:?}", Foo),
+    ///     format!("{:?}", Foo { foo: Box::new([1, 2].into_iter().copied()) }),
```

Comment:
🚩🚩🚩🚩🚩🚩

You do not want to use `.into_iter().copied()` here. This takes an autoref to `&[_]` and will break when we eventually add `impl IntoIterator for [_]`. Use `.iter()` instead.

Reply:
okies thanks!